examples/bigcommerce/components/product-cart.tsx

```diff
@@ -1,68 +1,243 @@
+import { Popover, Transition } from '@headlessui/react'
+import Image from 'next/image'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+
 import { useCart } from 'lib/cart-context'
 import { useProduct } from 'lib/product-context'
+import { Cart34, Minus28, Minus36, Plus28, Plus36, Spinner22, Spinner28 } from './icons'
+
+function formatPrice(value?: number) {
```

Comment:
Have you heard about the [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) API? I've never used it but it looks pretty cool!

Reply:
Going to pass on this add for now